test/sagemaker_tests/tensorflow/tensorflow1_training/requirements.txt

```diff
@@ -6,7 +6,7 @@ Pillow
 h5py
 tox
 flake8
-pytest==5.3.5
+pytest
 pytest-cov
 pytest-rerunfailures==9.0
 pytest-xdist
```

Comment:
If pytest-rerunfailures is pinned to a version, then pytest should also be pinned to a version.

Please upgrade that pinned version if necessary, but don't unpin it as it causes issues with test setup when other pytest packages are updated.

Reply:
I was using python2 to install the requirements for py2 images. so I had to unpin the version as 5.3.5 didn't support py2.

In this case the solution is to install the requirements and run tests using python3 for all the images.